### PR TITLE
feat(#736): Switch tabs with keyboard shortcut

### PR DESCRIPTION
**Pull request by ParamjotSingh5**
# Description

1. Registered keyboard events in Hotkeys/index.js
2. Added logic for replacing `state.activeTabUid` to switch active tab as per keyboard event.

### Contribution Checklist:

- [x] **The pull request only addresses one issue or adds one feature.**
- [x] **The pull request does not introduce any breaking changes**
- [x] **I have added screenshots or gifs to help explain the change if applicable.**
- [x] **I have read the [contribution guidelines](https://github.com/usebruno/bruno/blob/main/contributing.md).**
- [x] **Create an issue and link to the pull request.**

### CODE REVIEW

**Review by flof — APPROVED** [review]

File: packages/bruno-app/src/providers/ReduxStore/slices/tabs.js [hunk 1]
```diff
@@ -45,6 +45,34 @@ export const tabsSlice = createSlice({
...
+        }
+      } else if (direction === 'pagedown') {
+        if (activeTabIndex == state.tabs.length - 1) {
+          toBeActivatedTabIndex = 0;
+        } else {
+          toBeActivatedTabIndex = activeTabIndex + 1;
+        }
+      }
+
+      state.activeTabUid = state.tabs[toBeActivatedTabIndex].uid;
```
Comment (Jeremie-Chauvel) [inline packages/bruno-app/src/providers/ReduxStore/slices/tabs.js]: (dust = can be ignored) not my place to comment but I ended here after looking into ctrl-tab for Bruno, thanks for the implementation of the first shortcuts!

[I would recommend using the modulo operator here](https://www.joshwcomeau.com/javascript/modulo-operator/), that would simplify your code quite a bit  with taking care of the edge cases, something like:
```jsx
if (direction === 'pageup') {
  toBeActivatedTabIndex = (activeTabIndex - 1 + state.tabs.length) % state.tabs.length;
} else if (direction === 'pagedown') {
  toBeActivatedTabIndex = (activeTabIndex + 1) % state.tabs.length;
}

state.activeTabUid = state.tabs[toBeActivatedTabIndex].uid;

Comment (ParamjotSingh5) [reply packages/bruno-app/src/providers/ReduxStore/slices/tabs.js]: Sure @Jeremie-Chauvel, I will update here soon.

Thanks for sharing the blogpost 😇, It is good.

Comment (ParamjotSingh5) [reply packages/bruno-app/src/providers/ReduxStore/slices/tabs.js]: Hi @Jeremie-Chauvel, I pushed suggested modifications, you can review them. 😊

**Comment by ParamjotSingh5** [comment]
Hi @helloanoop, request you to review this PR, I am open to feedback or any modification.

**Comment by helloanoop** [comment]
Merged! Thanks @ParamjotSingh5 
This will be released in `v1.28.0`

My apologies for being so late on getting to this PR.

**Pull request merged**